### ISSUE TITLE
add repeat queue tickbox for continuous printing

### ISSUE THREAD

**Issue by rjelbert** [nukeem/continuousprint#14]
At the moment, printing finishes when you get to the end of the queue. If there was a tickbox in the queue management screen called repeat queue, it would be possible to print without stopping.

**Comment by nukeem** [comment]
Similar to issue #6 I'm still thinking about the best way to achieve this.

**Comment by freeflocw** [comment]
Yes, this would be very nice very useful,  that's what I am waiting for very long. Can I donate you a little via Paypal? fb@loxos.de

**Comment by dzid26** [comment]
I am thinking to have a number of copies of each model. 
Ideally each model should have option of custom clearing code associated to it. 

That would allow for to do small automated manufacturing
